pkg/changetracking/changetracking_api_utilities.go

```diff
@@ -0,0 +1,24 @@
+package changetracking
+
+import (
+	"time"
+
+	"github.com/newrelic/newrelic-client-go/v2/pkg/nrtime"
+)
+
+// DO NOT DELETE the following function - this is not covered by Tutone
+// but is needed to ensure proper conversion of timestamps
+func (input *ChangeTrackingDeploymentInput) CorrectTimestampMilliseconds() {
```

Comment:
Instead of making this a method on the `ChangeTrackingDeploymentInput`, this could be a nice little helper function and could be shared across all packages with a function name such as `getSafeTimestamp(...)`. Any thoughts on this?

Reply:
That actually makes sense; I did think of this but missed it somehow. Will do first thing tomorrow - will shift this out of the `changetracking` package as a helper function elsewhere :)

Reply:
@sanderblue @Avinash-Sanpala I've just moved this method to `utils` as `GetSafeTimestampWithMilliseconds` which operates on the `nrtime.EpochMilliseconds` type, but am still calling this within the current `CorrectTimestampMilliseconds()` method in `changetracking` (instead of shifting this entirely into `utils`) as I think it would be good to have the helper function use `nrtime.EpochMilliseconds`, which is common across packages. Let me know what you think about this change

Reply:
@pranav-new-relic I think its better to use the `utils.GetSafeTimestampWithMilliseconds` directly from `ChangeTrackingCreateDeployment` method of changetracking_api.go because I see there is one more type called `ChangeTrackingDeployment` which also has the timestamp field and I guess if this field needs to use this changetracking utility method `CorrectTimestampMilliseconds` sometime later, then again it has to add a similar method with the receiver type of `ChangeTrackingDeployment` like `func (input *ChangeTrackingDeployment) CorrectTimestampMilliseconds()` . Hence I think its better to directly call the `GetSafeTimestampWithMilliseconds` from changetracking api

Reply:
makes sense to me; thank you for the callout. @sanderblue @Avinash-Sanpala I just got rid of the additional caller function of utils - can you please take a look when free again